doc/decisions/gsettings_values.md

```diff
@@ -0,0 +1,52 @@
+# GSettings values
+
+## Issue
+
+`GSettings` uses `Glibs` `GVariant` type for it values. The `GVariant` type can hold many
+different variations of types but `Elektra` only distinguishes between strings
+and binary data.
+
+## Constraints
+
+ - saved values have to be readable by `GSettings` but also trough `Elektra` itself
+ - keep `Elektra` as `Glib` independent as possible
+ - values should be human readable
+
+## Assumptions
+
+ - `Elektras` strength is its modularity to provide multiple backends that decide how to save values
+ - `GVariants` can be printed as Strings and parsed back again if we know the type
+ - `GVariants` can be also serialized and deserialized
+ - representing `GVariants` values trough strings fulfills all three constraints
+
+## Considered Alternatives
+
+ - save serialized values as binarys,
+ - create a mapping between GVariants types and `spec` types and transform them when saving
+
+## Decision
+
+ - Use GVariants string representation
```

Comment:
Is the string representation of basic types (int, bool) compatible with Elektra's types?